### PR TITLE
Adding mandatory yaml files

### PR DESCRIPTION
**Pull request by lucasrf**

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  name: oauth2-proxy
+  annotations:
+    github.com/project-slug: philips-forks/oauth2-proxy
+    philips.com/techdocs-builder: local
+  description: TBD
+spec:
+  type: library
+  lifecycle: experimental
+  owner: pics
+  component: Reporting
```
Comment (FerdySchepers) [inline catalog-info.yaml:13]: component reporting changed into system please change to:
system: Reporting

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: oauth2-proxy
+  annotations:
```
Comment (FerdySchepers) [inline catalog-info.yaml:5]: add:
title: Oauth2-Proxy

File: catalog-info.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    github.com/project-slug: philips-forks/oauth2-proxy
+    philips.com/techdocs-builder: local
+  description: TBD
```
Comment (FerdySchepers) [inline catalog-info.yaml:8]: Maybe add some info from the readme file:
A reverse proxy and static file server that provides authentication using Providers (Google, GitHub, and others) to validate accounts by email, domain or group.

Note: This repository was forked from [bitly/OAuth2_Proxy](https://github.com/bitly/oauth2_proxy) on 27/11/2018. Versions v3.0.0 and up are from this fork and will have diverged from any changes in the original fork. A list of changes can be seen in the [CHANGELOG](https://github.com/philips-forks/oauth2-proxy/blob/master/CHANGELOG.md).

File: catalog-info.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: oauth2-proxy
+  annotations:
+    github.com/project-slug: philips-forks/oauth2-proxy
```
Comment (FerdySchepers) [inline catalog-info.yaml:6]: add:  backstage.io/techdocs-ref: dir:.

**Review by FerdySchepers — CHANGES_REQUESTED** [review]

**Review by FerdySchepers — APPROVED** [review]

File: mkdocs.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+site_name: Oauth2 Proxy
+site_author: Philips
+docs_dir: docs
+
```
Comment (FerdySchepers) [inline mkdocs.yml:4]: nav:
  - ...
plugins:
  - techdocs-core
  - awesome-pages
  
Should be added if we automatically wnt to generate the doc files

**Comment by l-lafin** [comment]
@lucasrf @FerdySchepers @sailinder, this repository belongs to the `philips-forks` and not `philips-internal`, and as a consequence it is not connected to the dev-portal. The files added here must be removed.

**Pull request merged**